src/components/LayoutBase/LayoutBase.tsx

```diff
@@ -16,6 +16,33 @@ type LayoutBaseProps = {
 export const LayoutBase = ({ children }: LayoutBaseProps) => {
   useNavbarStyles();
 
+  useEffect(() => {
+    const videoAskScript = document.createElement('script');
+    videoAskScript.innerHTML = `
+      window.VIDEOASK_EMBED_CONFIG = {
+        "kind": "widget",
+        "url": "https://www.videoask.com/fno6cfau6",
+        "options": {
+          "widgetType": "VideoThumbnailExtraLarge",
+          "text": "Hey!&#x1FAE1;",
```

Comment:
![Screenshot from 2023-08-17 09-41-25](https://github.com/vazco/forminer/assets/42475735/912f7d82-b04c-401e-8b76-0d9a47953f26)


I see a square instead of an icon. Is it working on your machine?

Reply:
![Screenshot 2023-08-17 at 09 54 26](https://github.com/vazco/forminer/assets/89641881/eda31283-e2a0-4599-b572-544a8e824ea2)

Yes, on mine it's working. I changed it to another code (HTML Dec instead of Hex) and maybe it will work on your machine too.